scripts/shared.yml

```diff
@@ -0,0 +1,4 @@
+document:
```

Comment:
What is the purpose of this file? Will it be a blog (or link to)?

Reply:
Every root project needs to have some basic information... normally this is stored in the `.spdx-builder.yaml` file, but now we're using a shared version, since we're not interested in the root projects, but in the dependencies.

When you want to later on process these `.spdx` files in [Bom-Base](https://github.com/philips-software/bom-base), you do need to add a specific `.spdx-builder` for each project.

I will add this in the blog.